### DISCUSSION TITLE
Disable suggestions / completion not working?

### DISCUSSION THREAD

**Discussion by DamienPetrilli** [zed-industries/zed#17528] [Feature Requests]
Hello,

I am trying to disable the completion / suggestion in Zed and I can't find anyway to get it working. I tried all the following options and suggestions keeps appearing while I type. Did I miss something or should I open a bug?
 
```JSON
{
"languages": {
    "Odin": {
      "show_inline_completions": false
    }
  },
  "show_inline_completions": false,
  "inline_completions": {
    "disabled_globs": [".md", ".odin"]
  },
}
```

Also I disabled all assistant so it's not them
```JSON
{
  "features": {
    "copilot": false,
    "inline_completion_provider": "none"
  },
  "collaboration_panel": {
    "button": false
  },
  "chat_panel": {
    "button": false
  },
  "assistant": {
    "enabled": false,
    "version": "2",
    "button": false
  },
  "assistant_v2": {
    "enabled": false
  }
}
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
